packages/flutter_tools/lib/src/android/gradle_errors.dart

```diff
@@ -669,3 +670,44 @@ final GradleHandledError sslExceptionHandler = GradleHandledError(
   },
   eventLabel: 'ssl-exception-tag-mismatch',
 );
+
+final RegExp _unsupportedClassFileMajorVersionPattern = RegExp(r'Unsupported class file major version ([0-9]+)');
```

Comment:
Since we are not doing anything with the group use a non captured group (I just learned about them from stuart. 

```suggestion
final RegExp _unsupportedClassFileMajorVersionPattern = RegExp(r'Unsupported class file major version (?:[0-9]+)');
```